File: spring-cloud/eurake-client/src/main/java/com/chen/clients/eurakeclient/controller/TestController.java
package com.chen.clients.eurakeclient.controller;

import org.springframework.beans.factory.annotation.Value;
import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RequestParam;
import org.springframework.web.bind.annotation.RestController;

@RestController
public class TestController {

    @Value("${server.port}")
    String port;

    @GetMapping("/get-string")
    public String getString(@RequestParam(name = "string") String string) {
        return string + "  port:" +port;
    }
}

File: spring-boot-logger/src/main/kotlin/com/chen/demo/controller/TestController.kt
package com.chen.demo.controller


import org.apache.logging.log4j.LogManager
import org.apache.logging.log4j.Logger
import org.springframework.http.ResponseEntity
import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.RestController
import java.lang.RuntimeException

@RestController
class TestController {

    val log: Logger = LogManager.getLogger(TestController::class.java)

    @GetMapping("/hello")
    fun getHello(): ResponseEntity<String> {

        log.debug( RuntimeException("successs-----------------------").toString())
        return throw RuntimeException("successs-----------------------")
//        return ResponseEntity.ok("success");

    }
}